packages/accounts-passwordless/passwordless_server.js

```diff
@@ -0,0 +1,246 @@
+import { Accounts } from 'meteor/accounts-base';
+import { getUserById, tokenValidator } from './server_utils';
+import { Random } from 'meteor/random';
+
+const ONE_HOUR_IN_MILLISECONDS = 60 * 60 * 1000;
```

Comment:
Just use `3600000` no need to perform calculations for something so basic that doesn't change.

Reply:
Big numbers usually are represented like this so it's easier to understand, you can see many places like this in our codebase and in general as well.

Reply:
Yes, I know. But I think that could be solved by a comment and no need to process math when it isn't needed. Just a suggestion.

Reply:
I think the name of the const provides all the info for readability that is needed. :man_shrugging: